README.md

```diff
@@ -63,6 +63,19 @@ _LTracker.push({
 });
 ```
 
+By default, tags are set once and are then immutable. To enable changing the tag with each log statement, set `mutableTags` to true when initializing:
+
+```Javascript
+_LTracker.push({
+  'logglyKey': 'your-customer-token',
+  'sendConsoleErrors' : true,
+  'tag' : 'tag1,tag2',
+  'mutableTags' : true
```

Comment:
Maybe we should default to false since this is the current behavior, and it might be unexpected that a tag field in the data would update the tag on the URL